Docs/svt-hevc_encoder_user_guide.md

```diff
@@ -495,6 +495,18 @@ If only TargetSocket is set, threads run on all the logical processors of socket
 
 If both LogicalProcessors and TargetSocket are set, threads run on 20 logical processors of socket 0. Threads guaranteed to run only on socket 0 if 20 is larger than logical processor number of socket 0.
 
+### 3. Header(VPS SPS PPS) insertion
+
+In some streaming usage, header needs to be inserted repeatedly into the bitstream. Otherwise, received stream may not be decodable.
```

Comment:
In some streaming use cases, headers need to be inserted repeatedly into the bitstream. Otherwise the received stream may not be decodable.